lib/rdoc/options.rb

```diff
@@ -355,18 +355,24 @@ class RDoc::Options
   # +--[no-]embed-mixins+ (Default is +false+.)
   attr_accessor :embed_mixins
 
+  ##
+  # Exclude default pattersn to exclude.
+  attr_reader :default_exclude
```

Comment:
Since the set of excluded patterns is called `@exclude` now, `@default_exclude` feels like would contain the value of `DEFAULT_EXCLUDE`. I think maybe `apply_default_exclude` expresses what this value does better?

Reply:
Updated.